apps/cowswap-frontend/src/modules/tradeFormValidation/services/validateTradeForm.ts

```diff
@@ -20,9 +20,14 @@ export function validateTradeForm(context: TradeFormValidationContext): TradeFor
     tradeQuote,
     account,
     isPermitSupported,
+    isInsufficientBalanceOrderAllowed,
   } = context
 
   const { inputCurrency, outputCurrency, inputCurrencyAmount, inputCurrencyBalance, recipient } = derivedTradeState
+  const isBalanceGreaterThan1Atom = inputCurrencyBalance
+    ? BigInt(inputCurrencyBalance.quotient.toString()) > BigInt(1)
+    : false
```

Comment:
The minimum of 1 atom is required by the backend.
Thus, true 0-balance orders are not allowed.